staff/victor-abadia/blog/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Blog</title>
+</head>
+
+<body>
+    <header>
+        <h1>My Blog</h1>
+    </header>
+    <br>
+    <nav>
+        <a href="#music">Music</a>
+        <a href="#films">Films</a>
+        <a href="#tv">Tv Shows</a>
+    </nav>
+    <br>
+    <article>
+        <h2>News</h2>
+        <section>
+            <h1>Title 1</h1>
+            <img src="http://www.thepopularapps.com/application/upload/ProfilePhoto/default-image-01.png">
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy
+                text of the printing and typesetting industry.</p>
+            <a href="post.html">read more</a>
+        </section>
+        <section>
+            <h1>Title 2</h1>
+            <img src="http://www.thepopularapps.com/application/upload/ProfilePhoto/default-image-08.png">
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum is simply dummy
+                text of the printing and typesetting industry.</p>
+            <a href="#">read more</a>
+        </section>
+    </article>
+    <br>
```

Comment:
br is not a semantic html tag in this content. You colud use it to separate paragraphs, but here is used for styling the page